### ISSUE TITLE
Stuck in Menu

### ISSUE THREAD

**Issue by ellie12134** [Kataiser/tf2-rich-presence#385]
The Rich Presence is stuck displaying that im on Menus despite after playing several matches

**Comment by Kataiser** [comment]
Yeah, that's a limitation of how I'm getting game info, which is technically solvable but I just haven't. Try increasing the "max kilobytes of console.log to scan" setting.

**Comment by ellie12134** [comment]
that didn't work at all my status still doesn't change even during a match

**Comment by ellie12134** [comment]
at this point im starting to doubt the setup/installer even installed properly but sadly that seems to be the only way to get this software
